packages/app/package-lock.json

```diff
@@ -0,0 +1,102 @@
+{
```

Comment:
we use yarn, there should be no package-lock.json files?

Reply:
Removed package json 👍